src/broker/Data.cc

```diff
@@ -345,6 +345,34 @@ struct val_converter {
 
 			return rval;
 			}
+		else if ( type->Tag() == TYPE_LIST )
+			{
+			// lists are just treated as vectors on the broker side.
+			auto lt = type->AsTypeList();
+			auto rval = make_intrusive<ListVal>(TYPE_ANY);
+			auto types = lt->GetTypes();
```

Comment:
```suggestion
			const auto& types = lt->GetTypes();
```
efficiency nitpick: a copy of the `vector` was unnecessarily created.